src/icons/fileIcons.ts

```diff
@@ -892,6 +892,7 @@ export const fileIcons: FileIcons = {
         { name: 'commitlint', fileNames: ['.commitlintrc', '.commitlintrc.js', 'commitlint.config.js', '.commitlintrc.json', '.commitlint.yaml', '.commitlint.yml'] },
         { name: 'buck', fileNames: ['.buckconfig'] },
         { name: 'dhall', fileExtensions: ['dhall', 'dhallb'] },
+        { name: 'sml', fileExtensions: ['sml', 'mlton', 'mlb', 'sig', 'fun', 'cm', 'lex', 'use', 'grm'] },
```

Comment:
Are these file extensions all related to SML or machine learning?

Reply:
All related to Standard ML and its compilers, environments etc

Reply:
Thanks for clarifying, looks good 👍